src/main/java/org/zalando/failsafeactuator/domain/FailsafeBreaker.java

```diff
@@ -0,0 +1,40 @@
+package org.zalando.failsafeactuator.domain;
+
+import com.codahale.metrics.Meter;
+import com.codahale.metrics.MetricRegistry;
+import lombok.Data;
+import net.jodah.failsafe.CircuitBreaker;
+
+@Data
+public class FailsafeBreaker extends CircuitBreaker {
+
+    private final Meter failureMeter;
+    private final Meter successMeter;
+    private final Meter overallMeter;
+    private String identifier;
+
+    public FailsafeBreaker(final MetricRegistry metricRegistry, final String identifier) {
+        this.identifier = identifier;
+        this.failureMeter = metricRegistry.meter(this.identifier+".failures");
+        this.successMeter = metricRegistry.meter(this.identifier+".success");
+        this.overallMeter = metricRegistry.meter(this.identifier+".overall");
+    }
+
+    @Override
+    public void recordResult(Object result) {
```

Comment:
this should be done with [event listeners](https://github.com/jhalterman/failsafe#event-listeners) rather than subclassing.